bluebrain/repo-bluebrain/packages/py-region-grower/package.py

```diff
@@ -7,26 +7,36 @@
 
 
 class PyRegionGrower(PythonPackage):
-    """Python library for space-aware neuron synthesis"""
+    """Python library for space-aware neuron synthesis."""
 
     homepage = "https://bbpgitlab.epfl.ch/neuromath/region-grower"
     git = "ssh://git@bbpgitlab.epfl.ch/neuromath/region-grower.git"
 
     version("develop", branch="main")
-    version("0.3.0", tag="region-grower-v0.3.0")
+    version("0.4.3", tag="region-grower-v0.4.3")
+    version("0.3.1", tag="region-grower-v0.3.1")
 
     depends_on("py-setuptools", type="build")
 
     depends_on("py-attrs@19.3.0:", type=("build", "run"))
     depends_on("py-click@7.0:", type=("build", "run"))
-    depends_on("py-dask+dataframe+distributed@2.15.0:2021.8.0,2021.9.0:", type=("build", "run"))
-    depends_on("py-diameter-synthesis@0.2.5:0.999", type=("build", "run"))
-    depends_on("py-morphio@3.0:3.999", type=("build", "run"))
-    depends_on("py-morph-tool@2.9.0:2.999", type=("build", "run"))
-    depends_on("py-neuroc@0.2.8:0.999", type=("build", "run"))
-    depends_on("py-neurom@3.0:3.999", type=("build", "run"))
-    depends_on("py-tns@2.5.0:2.999", type=("build", "run"))
+    depends_on(
+        "py-dask+dataframe+distributed@2.15.0:2021.8.0,2021.9.0:",
+        type=("build", "run"),
+        when="@0.3.1",
+    )
+    depends_on("py-dask+dataframe+distributed@2021.9:", type=("build", "run"), when="@0.4.3:")
+    depends_on("py-diameter-synthesis@0.4.1:0.999", type=("build", "run"), when="@0.3.1")
```

Comment:
For future reference: `:0.999` is equivalent to `:0`, we don't need the trailing 9s any longer :)

Reply:
Ah yeah, I realized that yesterday and removed a few ones but not all :)
Do you want me to remove them all in this PR?

Reply:
It can wait, next time you do a version bump you can fix more if you want

Reply:
Ok, I will try to remember next time :)
Thank you very much for your help in this PR!